csi/src/node/iscsiutil.rs

```diff
@@ -32,6 +32,19 @@ pub fn wait_for_path_to_exist(devpath: &str, max_retries: i32) -> bool {
     device_path.exists()
 }
 
+fn realpath(path: String) -> String {
```

Comment:
Nitpick but I think this function should be renamed because it's not exactly the same as realpath(3), itself provided by fs::canonicalize, but rather an internal iSCSI-specific variant.

Reply:
Fixed by https://github.com/openebs/Mayastor/pull/232/commits/974b40c85561e4771b75746145bf986f01355458